components/update_client/cobalt_slot_management.cc

```diff
@@ -329,12 +329,13 @@ bool CobaltQuickUpdate(
       continue;
     } else if (slot_candidate_version < installed_version &&
```

Comment:
Is the `slot_candidate_version` reliable?

Reply:
Yes, `slot_candidate_version` is a place holder of the highest version found. It starts with `1.0.1`. `installed_version` is the version read from `manifest.json` of the current iterated slot. It works for system image too. At the moment, we have to trust the `manifest.json` in the system image to be accurately reflecting the Evergreen version, as we don't have other ways to know the version before actually loading the binary.